proto/commonfate/access/v1alpha1/access_request.proto

```diff
@@ -40,26 +45,39 @@ service AccessRequestService {
   rpc CloseAccessRequest(CloseAccessRequestRequest) returns (CloseAccessRequestResponse) {}
 }
 
+enum RequestStatus {
+  REQUEST_STATUS_UNSPECIFIED = 0;
+
+  REQUEST_STATUS_PENDING = 1;
+
+  REQUEST_STATUS_ACTIVE = 2;
+
+  REQUEST_STATUS_CLOSED = 3;
+}
+
 message QueryAccessRequestsRequest {
   // The token for the next page.
   string page_token = 1;
   bool archived = 2;
   optional commonfate.entity.v1alpha1.Order order = 3;
-  optional commonfate.entity.v1alpha1.EID requested_by = 4;
   optional commonfate.entity.v1alpha1.EID closed_by = 5;
   optional commonfate.entity.v1alpha1.EID approved_by = 6;
   RequestStatus request_status = 7;
 
 }
 
-enum RequestStatus {
-  REQUEST_STATUS_UNSPECIFIED = 0;
-
-  REQUEST_STATUS_PENDING = 1;
+message QueryMyAccessRequestsResponse {
+  repeated AccessRequest access_requests = 1;
+  string next_page_token = 2;
+}
 
-  REQUEST_STATUS_ACTIVE = 2;
+message QueryMyAccessRequestsRequest {
+  // The token for the next page.
+  string page_token = 1;
+  bool archived = 2;
```

Comment:
Lets exclude this deprecated field on the new API, instead, you can just query for closed requests.

This archived field was from when we used the authz service which has now been removed.

```suggestion
```